tailwind.config.js

```diff
@@ -13,5 +17,5 @@ module.exports = {
         half: '50%'
       }
     }
-  }
+    }
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 4 spaces but found 6.](https://app.codacy.com/gh/web3community/devprotocol.xyz/pullRequest?prid=8426368)
```suggestion
    }
```